wdl/transforms/draft3/src/main/scala/wdl/draft3/transforms/linking/graph/package.scala

```diff
@@ -48,17 +50,36 @@ package object graph {
     }
   }
 
+  implicit val callElementUnlinkedValueGenerator: UnlinkedValueGenerator[CallElement] = new UnlinkedValueGenerator[CallElement] {
+    override def generatedValueHandles(a: CallElement, typeAliases: Map[String, WomType]): ErrorOr[Set[GeneratedValueHandle]] = {
```

Comment:
Thinking about including outputs... I think we're going to need to change this method to include a set of WOM `Callable`s as an argument.

~~Which is probably also going to mean we'll need to build up all of the `TaskDefinition` WOMs before we even begin to look at the workflow structure. Which actually, I think, probably does make sense (i.e. we need to know what our tasks are before we can start to build the workflow)~~ 
EDIT: I just looked at the rest of this PR, and in fact this seems to be what happens already, so it might just be as easy as changing the method signature